src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/TotalCountQueryString.feature.disabled

```diff
@@ -1,196 +1,192 @@
-    When sending a GET request to "/ed-fi/schools?totalCount=false" 
-            """
-            """
-    Then it should respond with 200
-    And header "totalCount" is not present
+        @ignore
+        Scenario: Validate totalCount Header is not included when equals to false
+            Given the following schools exist
+                  | schoolId  | nameOfInstitution             | gradeLevels         | educationOrganizationCategories |
+                  | 5         | School with max edorgId value | [ "Postsecondary" ] | [ "School" ]                    |
+                  | 255901001 | Grand Bend High School        | [ "Tenth grade" ]   | [ "School" ]                    |
+             When a GET request is made to "/ed-fi/schools?totalCount=false"
+             Then it should respond with 200
+              And the response headers does not include "totalCount"
 
-  @ignore
-  Scenario: Ensure that schools return the total count
-    Given these schools exist:
-            | schoolId     | nameOfInstitution                    | gradeLevels                  | educationOrganizationCategories      |
-            | 5            | School with max edorgId value        | [ "Postsecondary" ]          | [ "School" ]                         |
-            | 255901001    | Grand Bend High School               | [ "Tenth grade" ]            | [ "School" ]                         |
-    When sending a GET request to "/ed-fi/schools" 
-            """
-            """
-    Then it should respond with 200
-    And header "totalCount" is not present
+        @ignore
+        Scenario: Validate totalCount is not included when it is not present in the URL
+            Given the following schools exist
+                  | schoolId  | nameOfInstitution             | gradeLevels         | educationOrganizationCategories |
+                  | 5         | School with max edorgId value | [ "Postsecondary" ] | [ "School" ]                    |
+                  | 255901001 | Grand Bend High School        | [ "Tenth grade" ]   | [ "School" ]                    |
+             When a GET request is made to "/ed-fi/schools"
+             Then it should respond with 200
+              And the response headers does not include "totalCount"
 
+        @ignore
+        Scenario: Ensure clients can GET information changing the casing of the query value to be all lowercase
+            Given the following schools exist
+                  | schoolId | nameOfInstitution             | gradeLevels         | educationOrganizationCategories |
+                  | 5        | School with max edorgId value | [ "Postsecondary" ] | [ "School" ]                    |
+             When a GET request is made to "/ed-fi/schools?nameOfInstitution=school+with+max+edorgid+value"
+             Then it should respond with 200
+              And the response body includes "nameOfInstitution: School with max edorgId value"
```

Comment:
I just noticed that this has nothing to do with TotalCount, which is the feature. But we can refactor that later. I put a note into DMS-209 to address this.